graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -141,6 +145,8 @@ def run_cypher(
             connectivity_retry_config = Neo4jQueryRunner.ConnectivityRetriesConfig()
         self._verify_connectivity(database=database, retry_config=connectivity_retry_config)
 
+        self._driver.session()
```

Comment:
what is this doing?

Reply:
Ah this is some leftover code. Will remove it